template/.devcontainer.json

```diff
@@ -0,0 +1,10 @@
+{
+    "forwardPorts": [
+        5000, // Flask
+        5900, // VNC server
+        6081, // VNC client
+        8080, // http-server
+        8082  // phpLiteAdmin
+    ],
+    "image": "ghcr.io/cs50/codespace"
+}
```

Comment:
Since this template contains so few keys, and we only access it via `/.devcontainer.json`, should `app.py` just generate the JSON dynamically?

Reply:
That works. Though seems easier to maintain the configuration file in one place? If we want to tweak student's `.devcontainer.json` file in the future, we can just do it here.

Reply:
Hmm, could we perhaps do this in cs50.dev:

`cs50.dev/build/.devcontainer.json` refers to the configuration file we used to build the image
`cs50.dev/template/.devcontainer.json` refers to the configuration file we put in student's repo

Reply:
Maybe `cs50.dev/dist/.devcontainer.json` would be more conventional in that model?

Reply:
`cs50.dev/dist/.devcontainer.json` for template configuration file and `cs50.dev/.devcontainer.json` remains the same?

Reply:
Maybe this would be clearest within the repo?

* `cs50.dev/.devcontainer/build/devcontainer.json` refers to the configuration file we used to build the image
* `cs50.dev/.devcontainer/dist/devcontainer.json` refers to the configuration file we put in student's repo